File: LeetCode-Programs/Sort Array By Parity.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        new_list=[]
        for i in nums:
            if i%2==0:
                new_list.append(i)
        for j in nums:
            if j%2!=0:
                new_list.append(j)
        return new_list
        

File: LeetCode-Programs/Sort Array By Parity.cpp
class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& nums) {
        vector<int> vec;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]%2==0)
                vec.push_back(nums[i]);
        }
        for(int j=0;j<nums.size();j++)
        {
            if(nums[j]%2!=0)
                vec.push_back(nums[j]);
        }
        return vec;
    }
};
